src/components/Dropdown/V2/Dropdown.Item.tsx

```diff
@@ -238,11 +238,14 @@ export class Item extends React.PureComponent<Props> {
 
   setNodeRef = node => {
     this.node = node
-    this.props.innerRef(node)
+    this.props.ref(node)
   }
   setActionNodeRef = node => (this.actionNode = node)
   setWrapperNodeRef = node => (this.wrapperNode = node)
-  setMenuNodeRef = node => (this.menuNode = node)
+  setMenuNodeRef = node => {
+    console.log('menuRef', node)
```

Comment:
console.log here

Reply:
actually many, wip I know....

Reply:
leftovers from debugging why the dropdown was not rendering correctly (actually it was because styles where render in both styled-component and emotion, took me 2 days to figure that out 🤦‍♂ )

Reply:
That Dropdown causes a tonne of problems.